hoprd/rest-api/src/session.rs

```diff
@@ -22,56 +31,94 @@ use crate::{ApiErrorStatus, InternalState, BASE_PATH};
         "path": {
             "Hops": 1
         },
-        "port": 0
+        "target": "localhost:8080",
+        "listen_host": "127.0.0.1:10000",
+        "capabilities": ["Retransmission", "Segmentation"]
```

Comment:
Why does it make sense to allow setting capabilities when the protocol os `tcp`?

Reply:
For debugging, the default for TCP are all caps, default for UDP is no caps. 
This is only there to allow override when debugging.